pages/api/event-save-changes.ts

```diff
@@ -0,0 +1,27 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getSession } from 'next-auth/react';
+import dbConnect from '../../middleware/database';
+import Event from '../../models/event';
+import { EventData } from '../../types/database';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse<any>) {
+	const session = await getSession({ req });
+	if (session?.userType !== 'ORGANIZER') return res.status(403).send('Forbidden');
+
+	await dbConnect();
+
+	switch (req.method) {
+		case 'POST':
+			const events: EventData[] = req.body as EventData[];
+			console.log(events);
+
+			events.forEach(async event => {
+				const { _id, nfcPoints } = event;
+				await Event.findByIdAndUpdate(_id, { nfcPoints });
```

Comment:
instead of doing a `forEach(...)` and doing an `await Event.findByIdAndUpdate(...)` for each iteration, you can make a batch operation all at once

for reference (i did not check if this works) 
<img width="850" alt="image" src="https://github.com/VandyHacks/witness/assets/58854510/d2dfb6ef-782a-4826-a9da-8a373c97c219">

Reply:
ok

Reply:
![image](https://github.com/VandyHacks/witness/assets/58854510/6ea74390-9e02-4eae-99c4-6ac7af8e6416)